src/performance.js

```diff
@@ -173,6 +175,15 @@ export class Performance {
     }
   }
 
+  /**
+   * @param {string} label  The variable name as it will be reported.
+   * @param {number} value The value in milliseconds that should be ticked.
+   */
+  tickDelta(label, value) {
+    this.tick('_' + label, undefined, 0);
+    this.tick(label, '_' + label, value);
```

Comment:
this makes more sense now that just doing `tick('label')` since i know the underlying library will use `now()`